Description: Add the missing dependency management entries that should be
 inherited from the ignored parent pom. This fixes a Maven warning displayed
 when invoking the maven-compiler-plugin.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/pom.xml
+++ b/pom.xml
@@ -105,6 +105,11 @@
         <artifactId>hamcrest</artifactId>
         <version>2.2</version>
       </dependency>
+      <dependency>
+        <groupId>org.codehaus.plexus</groupId>
+        <artifactId>plexus-utils</artifactId>
+        <version>3.0.22</version>
+      </dependency>
     </dependencies>
   </dependencyManagement>
   <dependencies>
